UndertaleModLib/Models/UndertaleRoom.cs

```diff
@@ -1527,6 +1554,12 @@ public void Unserialize(UndertaleReader reader)
                 Rotation = reader.ReadSingle();
             }
 
+            public static UndertaleString GenerateRandomName(UndertaleData data)
+            {
+                // The same format as in "GameMaker Studio: 2".
```

Comment:
No need for quotation marks.

Reply:
There's a `:` character in the title, so I think they should stay here.